annotations.md

```diff
@@ -22,4 +22,4 @@ This specification defines the following annotation keys, intended for but not l
 * **org.opencontainers.image.homepage** URL to find more information on the image (string, a URL with scheme HTTP or HTTPS)
 * **org.opencontainers.image.documentation** URL to get documentation on the image (string, a URL with scheme HTTP or HTTPS)
 * **org.opencontainers.image.source** URL to get source code for the binary files in the image (string, a URL with scheme HTTP or HTTPS)
-* **org.opencontainers.image.ref.name** Name of the reference (string)
+* **org.opencontainers.image.ref.name** Name of the reference for a target (string). Only valid when on descriptor within an index.json in [image layout](image-layout.md).
```

Comment:
If we don't see this being used elsewhere then yeah, it may make sense to move it closer to the `application/vnd.oci.image.index.v1+json` media type. However in tandem we should tweak the wording of this document to be clear that these are not the only annotations.